### PR TITLE
Unit tests improvement and addition of edmFileInPath

### PR DESCRIPTION
**Pull request by smuzaffar**
- Convert test to use `<test ...command=""/>` instead of FW utility driver
- Make use of `edmFileInpath` to find the data file

### CODE REVIEW

**Comment by tvami** [comment]
@cmsbuild , please test

File: CondTools/Hcal/test/pfcuts_db_io_test.csh [hunk 1]
```diff
@@ -1,10 +1,7 @@
-#!/bin/csh
+#!/bin/bash -ex
```
Comment (smuzaffar) [inline CondTools/Hcal/test/pfcuts_db_io_test.csh:1]: I will rename it to `pfcuts_db_io_test.sh`

**Comment by smuzaffar** [comment]
please test

**Comment by tvami** [comment]
+db

**Comment by makortel** [comment]
+core

**Comment by perrotta** [comment]
+1

**Pull request merged**